src/ray/raylet/scheduling/cluster_resource_scheduler_interface.h

```diff
@@ -71,6 +72,13 @@ class ClusterResourceSchedulerInterface {
   /// fields used.
   virtual void FillResourceUsage(rpc::ResourcesData &data) = 0;
 
+  /// Populate a UpdateResourcesRequest. This is inteneded to update the
+  /// resource totals on a node when a custom resource is created or deleted
+  /// (e.g. during the placement group lifecycle).
+  ///
+  /// \param Output parameter. Fills out all fields.
```

Comment:
Nit: Return docstring item instead of parameter.
```suggestion
  /// \return Filled out resource map.
```